### PR TITLE
Automation fixes for rhosp failures for stable/4.2

### PR DESCRIPTION
**Pull request by nasharatm**

### CODE REVIEW

File: tempest/util/query_data.py [hunk 1]
```diff
@@ -348,8 +348,10 @@ def get_snapshot_restore_id(snapshot_id):
     try:
         conn = db_handler.dbHandler()
         cursor = conn.cursor()
+        #get_snapshot_restore_id = ("select id from restores where snapshot_id='" +
```
Comment (savitha-p) [inline tempest/util/query_data.py]: Remove the commented code.

File: tempest/api/workloadmgr/restore/test_selectiverestore_defaultvalues.py [hunk 2]
```diff
@@ -193,6 +193,7 @@ def test_selectiverestore_defaultvalues(self):
             md5_sum_after_selective_restore = {}
             md5sum_after_restore_list = []
             for floating_ip in self.restored_vm_floating_ips:
+                self.md5_sum_after_selective_restore[str(floating_ip)] = {}
```
Comment (SuhasTA4) [inline tempest/api/workloadmgr/restore/test_selectiverestore_defaultvalues.py]: @nasharatm  -    At line no. 193 we are declaring ==> md5_sum_after_selective_restore = {} and copying data inside the same, at line no. 203 ==>  md5_sum_after_selective_restore[str(floating_ip)][str(
                        mount_point)] = temp_md5sum
                        
Then, why we are using (line no. 196) ==> self.md5_sum_after_selective_restore[str(floating_ip)] = {}. What is the use ?
Either, we need to copy it to self.md5_sum_after_selective_restore or we need to use just md5_sum_after_selective_restore[str(floating_ip)] = {} at line no. 196

Please correct me, if I missed anything/ misinterpreted anywhere....

**Review by savitha-p — APPROVED** [review]

File: tempest/util/query_data.py (hunk 1 above)
Comment (nasharatm) [reply tempest/util/query_data.py]: done

**Review by SuhasTA4 — CHANGES_REQUESTED** [review]

File: tempest/api/workloadmgr/restore/test_selectiverestore_defaultvalues.py (hunk 2 above)
Comment (nasharatm) [reply tempest/api/workloadmgr/restore/test_selectiverestore_defaultvalues.py]: done changes

**Review by SuhasTA4 — APPROVED** [review]

**Pull request merged**